src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the `Person` component is correctly defined in `./components/Person/Person` and can handle all the properties being passed, including cases where some properties might be missing, like `age` for `olya`.